src/libasr/codegen/asr_to_wasm.cpp

```diff
@@ -13,6 +13,8 @@
 #include <libasr/exception.h>
 #include <libasr/asr_utils.h>
 
+#include <lfortran/pickle.h>
```

Comment:
You have to comment it out, since you can't use any LFortran specific header files in `libasr`, as it will fail in LPython:
```suggestion
// #include <lfortran/pickle.h>
```

Reply:
Thank you for the feedback. I just pushed a commit with this change.